pytorch_lightning/trainer/trainer.py

```diff
@@ -396,6 +397,7 @@ def __init__(
         self.interrupted = False
         self.should_stop = False
         self.running_sanity_check = False
+        self.state = TrainerState.INITIALIZING
```

Comment:
Here you set trainer's `self.state` to `TrainerState.INITIALIZING`...(see next comment)